weaviate/client.py

```diff
@@ -88,6 +89,9 @@ def __init__(
             Can be used to set OpenAI/HuggingFace keys. OpenAI/HuggingFace key looks like this:
                 {"X-OpenAI-Api-Key": "<THE-KEY>"}, {"X-HuggingFace-Api-Key": "<THE-KEY>"}
             by default None
+        wait_for_weaviate : int or None
+            Describes how long the client will wait for weaviate to start in seconds.
+            If wait_for_weaviate is None the client will not wait at all. Waits for 30s by default.
```

Comment:
What do you think about this?

How long the client will wait for weaviate to start before raising XXX.
If None the client will not wait at all. Default timeout is 30s.